app/Controllers/feedController.php

```diff
@@ -178,6 +178,7 @@ public function addAction(): void {
 			$proxy_type = Minz_Request::paramString('proxy_type');
 			$request_method = Minz_Request::paramString('curl_method');
 			$request_fields = Minz_Request::paramString('curl_fields', true);
+			$headers = preg_split('/\r\n|\r|\n/', Minz_Request::paramString(key: 'http_headers'));
```

Comment:
```suggestion
			$headers = Minz_Request::paramTextToArray('http_headers');
```

Please try that everywhere